openedx/core/djangoapps/user_api/accounts/permissions.py

```diff
@@ -13,3 +14,16 @@ class CanDeactivateUser(permissions.BasePermission):
     """
     def has_permission(self, request, view):
         return request.user.has_perm('student.can_deactivate_users')
+
+
+class CanRetireUser(permissions.BasePermission):
```

Comment:
@mikedikan @dianakhuang You guys will be able to use this permission for the LMS endpoints you are working on.